### PR TITLE
Small updates to scripts

### PR DESCRIPTION
**Pull request by adairrr**
- Adds manual deployment script for Abstract + Polytone
- Ensures that AbstractIBC contracts are registered
- Temporarily adds rollkit testnet

### Checklist

- [x] CI is green.
- [ ] Changelog updated.

### CODE REVIEW

**Review by cyberhoward — APPROVED** [review]

File: interchain/scripts/src/bin/connect_ibc.rs [hunk 1]
```diff
@@ -1,38 +1,26 @@
...
-
     let chains = vec![
+        (ROLLKIT_TESTNET, None),
+        (OSMO_5, None),
         (JUNO_1, None),
         (PHOENIX_1, None),
         (ARCHWAY_1, None),
-        (neutron, None),
-        (OSMOSIS_1, Some(std::env::var("OSMOSIS_MNEMONIC")?)),
+        (NEUTRON_1, None),
```
Comment (Buckram123) [inline interchain/scripts/src/bin/connect_ibc.rs:22]: nice

File: interchain/scripts/src/bin/full_deploy.rs [hunk 2]
```diff
@@ -134,15 +138,18 @@ fn main() {
...
-        .iter()
-        .map(|n| parse_network(n).unwrap())
-        .collect::<Vec<_>>();
+    // let networks = args
+    //     .network_ids
+    //     .iter()
+    //     .map(|n| parse_network(n).unwrap())
+    //     .collect::<Vec<_>>();
+
+    let networks = vec![ROLLKIT_TESTNET];
```
Comment (Buckram123) [inline interchain/scripts/src/bin/full_deploy.rs:152]: I guess we should revert changes of this file

File: interchain/scripts/src/bin/manual_deploy.rs
Comment (Buckram123) [inline interchain/scripts/src/bin/manual_deploy.rs:1]: nice!

**Review by Buckram123 — COMMENTED** [review]
Let's do a small cleanup and LGTM

**Review by Buckram123 — APPROVED** [review]

**Pull request merged**